docs/source/tutorials/qaoa_quantum_dropout.ipynb

```diff
@@ -0,0 +1,1040 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    QAOA_vvag = K.jit(K.vvag(QAOAansatz_iso, argnums=0, vectorized_argnums=0), static_argnums=(1, 2, 3), jit_compile=True if backend == 'JaxBackend' else False)
</pre><p>jit_compile has no effect for jax backend, so we can directlt omit this argument as its default False</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tencent-quantum-lab/tensorcircuit/pull/157/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>